app/assets/locales/de/nutritionData.json

```diff
@@ -0,0 +1,17 @@
+{
+  "nutritionTitle": "Nährwertangaben",
+  "energy": "Energie",
+  "energyUnit": "kcal",
+  "protein": "Proteine",
+  "proteinUnit": "g",
+  "carbohydrates": "Kohlenhydrate",
+  "carbohydratesUnit": "g",
+  "sugar": "davon Zucker",
+  "sugarUnit": "g",
+  "fat": "Fett",
+  "fatUnit": "g",
+  "saturatedFat": "davon gesättigt",
+  "saturatedFatUnit": "g",
+  "salt": "Salz",
+  "saltUnit": "g"
```

Comment:
Wouldn't it be better practice to only have one entry for grams? This way, if one unit changes in the future, a change has to be made in the source code but (possibly) not here. This is also kind of redundant.

Reply:
I think, that it is useful to keep the different units separated. Anyway, I don't think that this is a big issue nontheless, since it is highly unlikely that the nutrition data table set by the EU will change much in future.